apps/fz_http/lib/fz_http_web/plug/require_local_authentication.ex

```diff
@@ -0,0 +1,16 @@
+defmodule FzHttpWeb.Plug.RequireLocalAuthentication do
+  use FzHttpWeb, :controller
+
+  def init(opts), do: opts
+
+  def call(conn, _opts) do
+    if FzHttp.Configurations.get!(:local_auth_enabled) do
+      conn
+    else
+      conn
+      |> put_resp_content_type("text/plain")
+      |> send_resp(401, "Local auth disabled")
```

Comment:
I feel like 401 `Unauthorized ` is not the correct status response here, I guess It should be either 404 `Not Found` or 403 `Forbidden`:
```suggestion
      |> send_resp(403, "Local auth disabled")
```

Reply:
Good catch, agreed